ui/pages/bridge/prepare/prepare-bridge-page.tsx

```diff
@@ -218,8 +305,10 @@ const PrepareBridgePage = () => {
             testId: 'to-amount',
             readOnly: true,
             disabled: true,
-            value: toAmount?.toString() ?? '0',
-            className: toAmount ? 'amount-input defined' : 'amount-input',
+            value: activeQuote?.toTokenAmount?.amount.toFixed(2) ?? '0',
```

Comment:
are we sure we want to shorten to two decimal places? I'm assuming it's very long now, but do we have a standard on how much we should show of the `toTokenAmount`?

Reply:
Good question - there's a generic standard for extension, where we show something like `<0.000001`. But we may use a different default precision after the Bridge re-design so I'll hold off on truncating this number until then